pkg/kubelet/cm/devicemanager/checkpoint/checkpoint.go

```diff
@@ -18,22 +18,26 @@ package checkpoint
 
 import (
 	"encoding/json"
+	"hash/fnv"
+	"strings"
 
+	"github.com/davecgh/go-spew/spew"
 	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/kubernetes/pkg/kubelet/checkpointmanager"
 	"k8s.io/kubernetes/pkg/kubelet/checkpointmanager/checksum"
+	"k8s.io/kubernetes/pkg/kubelet/cm/devicemanager/device"
 )
 
-// DeviceManagerCheckpoint defines the operations to retrieve pod devices
+// DeviceManagerCheckpoint defines the operations to retrieve pod device
```

Comment:
Another Commit seems to have the same change?

Reply:
This is an unnecessary change and has been undone